i2c-flash.c

```diff
@@ -22,12 +22,20 @@ ISR(TWI_vect) {
 
 	uint8_t offset;
 
-	switch (TWSR) {
+	switch (TW_STATUS) {
```

Comment:
Good catch, `TW_STATUS` is definitely more correct here. The datasheet says we need to mask out the prescaler bits, and I hadn't done that.